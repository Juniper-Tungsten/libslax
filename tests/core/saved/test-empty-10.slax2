/* Machine Crafted with Care (tm) by slaxWriter */
version 1.1;

output-method xml {
    indent "yes";
}

match / {
    <op-script-results> {
        <output> format-number(1.15, "####.#");
        <output> format-number(10.15, "####.#");
        <output> format-number(100.15, "####.#");
        <output> format-number("100.15", "####.#");
        <output> format-number(1000.15, "####.#");
        <output> format-number("100.15", "####.################");
        <output> format-number(100.15, "####.################");
        <output> format-number(number("100.15"), "####.################");
    }
}
