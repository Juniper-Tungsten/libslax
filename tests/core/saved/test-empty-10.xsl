<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="1.0">
  <xsl:output method="xml" indent="yes"/>
  <!-- See https://bugzilla.gnome.org/show_bug.cgi?id=629325 -->
  <xsl:template match="/">
    <op-script-results>
      <output>
        <xsl:value-of select="format-number(1.15, &quot;####.#&quot;)"/>
      </output>
      <output>
        <xsl:value-of select="format-number(10.15, &quot;####.#&quot;)"/>
      </output>
      <output>
        <xsl:value-of select="format-number(100.15, &quot;####.#&quot;)"/>
      </output>
      <output>
        <xsl:value-of select="format-number(&quot;100.15&quot;, &quot;####.#&quot;)"/>
      </output>
      <output>
        <xsl:value-of select="format-number(1000.15, &quot;####.#&quot;)"/>
      </output>
      <output>
        <xsl:value-of select="format-number(&quot;100.15&quot;, &quot;####.################&quot;)"/>
      </output>
      <output>
        <xsl:value-of select="format-number(100.15, &quot;####.################&quot;)"/>
      </output>
      <output>
        <xsl:value-of select="format-number(number(&quot;100.15&quot;), &quot;####.################&quot;)"/>
      </output>
    </op-script-results>
  </xsl:template>
</xsl:stylesheet>
