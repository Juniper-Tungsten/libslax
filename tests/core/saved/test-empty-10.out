<?xml version="1.0"?>
<op-script-results>
  <output>1.2</output>
  <output>10.2</output>
  <output>100.1</output>
  <output>100.2</output>
  <output>1000.2</output>
  <output>100.1500000000000057</output>
  <output>100.1499999999999915</output>
  <output>100.1500000000000057</output>
</op-script-results>
